Rules

```diff
@@ -12,6 +12,14 @@
 #   “content/about.html”). To select all children, grandchildren, … of an
 #   item, use the pattern “/about/**/*/”.
 
+require 'nanoc-redirector'
+
+postprocess do
+  @items.each do |item|
+    NanocRedirector::RedirectFrom.process(item, item.identifier.without_ext, output_dir: 'content')
```

Comment:
The HTML files should not be in the `content` directory, they should be in the `output` directory with the other files.